cookbook/core/control_flow/run_conditions.py

```diff
@@ -47,10 +47,10 @@ def double(n: float) -> float:
 def multiplier(my_input: float) -> float:
     return (
         conditional("fractions")
-        .if_((my_input >= 0.1) & (my_input <= 1.0))
-        .then(double(n=my_input))
-        .else_()
-        .then(square(n=my_input))
+            .if_((my_input >= 0.1) & (my_input <= 1.0))
```

Comment:
What's the reason for the extra indentation?

Reply:
I think this was done by `black` auto-format